src/FacultyDirectory/Helpers/AuthorizationHandler.cs

```diff
@@ -21,9 +21,9 @@ public AuthorizationHandler(ApplicationDbContext dbContext)
                                                        AdminRequirement requirement)
         {
             var username = context.User.Identity.Name;
-            var user = await dbContext.Users.Where(u => u.Username == username).FirstOrDefaultAsync();
+            var user = await dbContext.Users.AnyAsync(u => u.Username == username);
 
-            if (user != null)
+            if (user)
```

Comment:
I'd either move the await into the if and toast the user value, or change the variable name to userExists or similar.

Reply:
yea probably worth changing the variable to something like `isUserAdmin`